autoload/sqhell.vim

```diff
@@ -118,7 +131,19 @@ function! sqhell#GetTableName()
     wincmd p
     let table = expand('<cword>')
     let tmp_db = mysql#GetDatabaseName()
-    let db = substitute(tmp_db, '^\s*\(.\{-}\)\s*$', '\1', '')
+    let db = sqhell#TrimString(tmp_db)
     execute savewin . "wincmd w"
     return [table, db]
 endfunction
+
+function! sqhell#CreateCSVFromRow(row)
```

Comment:
Again, I think this is provider specific as it relies on '|'s being present in the output.
Postgres or SQLite for example may never show pipes so this should go into `autoload/mysql.vim`

Reply:
Ok, I'll move it to the correct file.